apps/verify/models/cookie.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const uuidv4 = require('uuid/v4');
+const read = require('./read-cookie');
+const write = require('./write-cookie');
+
+const readUuid = async(uuid) => {
+  return await read(uuid);
+};
+
+const writeCookie = async(data) => {
+  data.uuid = uuidv4();
```

Comment:
This will overwrite `.uuid` if it already exists. Is that okay?

Reply:
yeah it's OK i plan to create it here

Reply:
it doesn't exist